homeassistant/components/unifi/sensor.py

```diff
@@ -18,35 +22,22 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up sensors for UniFi integration."""
     controller = get_controller_from_config_entry(hass, config_entry)
-    sensors = {}
+    controller.entities[DOMAIN] = {RX_SENSOR: set(), TX_SENSOR: set()}
 
     option_allow_bandwidth_sensors = controller.option_allow_bandwidth_sensors
```

Comment:
Can we remove all the non local stored options? Why do we need to compare the new option state with the old option state?

I'm thinking we can just let the options updated signal reach `add_entities`, if the option state is active. There will not be more than one type of entity per mac added anyway since we keep track of that with the type to macs map.

Reply:
At one point Paulus wanted to make sure to only update on changed options. There where also some issues with creating removing entities. But I will reevaluate the need for it

Reply:
At least for sensors its not a problem, I will continue evaluating this on the other platforms